presentation/src/main/java/com/knocklock/presentation/lockscreen/LockScreenViewModel.kt

```diff
@@ -26,7 +30,7 @@ class LockScreenViewModel constructor(context: Context) : ViewModel() {
     }
     private fun getNotificationList() {
         viewModelScope.launch {
-            notificationRepository.getNotificationList().collect { list ->
+            getNotificationUseCase().collect { list ->
                 val notificationList = list.map {
                     it.toModel()
```

Comment:
```suggestion
                val notificationList = list.map {
                    it.toModel()
                _notificationList.update { NotificationUiState.Success(notificationList) }
```

저는 개인적으로 StateFlow에서 list를 받을 때 1회성 업데이트가  아니라면 update를 사용하고 있습니다. 
결과적으로 동작은 같으나 값을 update는 값을 원자적을 비교하고 Thread Safe하는 특징이 있어서 사용하고 있는데요 참고해 주시면 감사하겠습니다  ^^
[매쉬업꺼](https://mashup-android.vercel.app/mashup-11th/koba/CompareAndSetInMutableStateFlow/AtomicOnMutableStateFlow/)
[제꺼^^](https://kimansu.medium.com/27-%EC%95%88%EB%93%9C%EB%A1%9C%EC%9D%B4%EB%93%9C-%EA%B3%B5%EB%B6%80-sharedflow-stateflow-8cc21167b8a8)

Reply:
읽어보고 고려하겠습니다 ~^^

Reply:
update가 비동기 상황에서 데이터의 원자성이 보장해줘서 좋아보이지만 내부코드를 보면 lock을 걸어서 사용하는 방식이기 때문에 잘 고려해서 사용하면 좋을 것 같아요!

제 생각에는 위 케이스의 경우에서는 값을 이전 값이 중요하지 않기 때문에 update는 안해도 될 것 같아요 :)

Reply:
현재 케이스에서는 update를 사용하지 않고, 단일 StateFlow에 접근하는 function이 여러개일 경우에 update를 사용할께요 감사합니다 @jaeryo2357 @KimHance